haystack/components/audio/whisper_local.py

```diff
@@ -16,7 +16,20 @@
 
 
 logger = logging.getLogger(__name__)
-WhisperLocalModel = Literal["tiny", "small", "medium", "large", "large-v2"]
+WhisperLocalModel = Literal[
```

Comment:
```suggestion
WhisperLocalModel = Literal[
    "base",
    "base.en",
    "large",
    "large-v1",
    "large-v2",
    "large-v3",
    "medium",
    "medium.en",
    "small",
    "small.en",
    "tiny",
    "tiny.en"
]
```

nit: just having the models listed alphabetically ordered